src/pages/data-management/index.jsx

```diff
@@ -82,14 +75,16 @@ const DataManagementPage = ({ route }) => {
 
     // Right now we have one experiment per project, so we can just load the experiment
     // This has to be changed when we have more than one experiment
-    const activeExperimentId = activeProject.experiments[0];
+    const activeExperimentId = projectsList[activeProjectUuid].experiments[0];
```

Comment:
this will fail if `activeProjectUuid` or `projectsList[activeProjectUuid]` or `projectsList[activeProjectUuid].experiments` is none. Can you add checks for this?

Reply:
added checks for the existence of `.experiments` and then `.experiments[0]`